### DISCUSSION TITLE
Static Site With Cloudflare

### DISCUSSION THREAD

**Discussion by sureshkumargondi** [EasyEngine/easyengine#1512] [Support Requests]
Hey Hi! 

Little help needed with this

I have deployed the static site HTML on DO using EasyEngine... I have enabled the cloudflare CDN after few months 

I have 301 redirect because of this setup. 

Screenshot of webpagetest.org 

[https://p22.p4.n0.cdn.getcloudapp.com/items/yAuDRm5Y/c495905d-0a70-4a71-81a4-9bf1ba2a8f75.jpg?v=9ef9afb5e06a2ef7196cf74717e74804](https://p22.p4.n0.cdn.getcloudapp.com/items/yAuDRm5Y/c495905d-0a70-4a71-81a4-9bf1ba2a8f75.jpg?v=9ef9afb5e06a2ef7196cf74717e74804)

The present setup is causing the issue of 301. is there a way setup on an IP address.. and only the cloudflare can access it? 

URL of the webpage I'm taking about 

[https://donateers.org/il/chicago/car-donation/](https://donateers.org/il/chicago/car-donation/)

Thank you
Suresh

**Comment by dhsathiya** [comment]
This might help: https://stackoverflow.com/a/8442408

The example is given in the above answer blocks whole site access.
For the URL that you gave, you have to update the location block in the Site-Nginx configuration.

Eg,

```nginx
location /il/chicago/car-donation/ {
    ...
}
```
